src/index.html

```diff
@@ -1,13 +1,423 @@
+          <h2 class="page__section-title">
+            Compare bikes
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="compare__item grid__item grid__item--1-4">
+              <article
+                class="card"
+              >
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/sporty4.jpg"
+                    alt="sporty4"
+                  >
+                </div>
+                <h3 class="card__title">Sporty 4</h3>
+                <p class="card__paragraph">
+                  The iconic frame brought to a new performance
+                  height as a sporty, active ride.
+                </p>
+                <span class="card__price">$ 2 590</span>
+              </article>
+            </div>
+
+            <div class="compare__item grid__item grid__item--5-8">
+              <article class="card">
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/rideInTownST.jpg"
+                    alt="sporty4"
+                  >
+                </div>
+                <h3 class="card__title">Ride in town ST</h3>
+                <p class="card__paragraph">
+                  An open frame for an upright riding position
+                  as the most comfortable ride in town.
+                </p>
+                <span class="card__price">$ 2 590</span>
+              </article>
+            </div>
+
+            <div class="compare__item grid__item grid__item--9-12">
+              <article class="card">
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/agileRide3.jpg"
+                    alt="sporty4"
+                  >
+                </div>
+                <h3 class="card__title">Agile ride 3</h3>
+                <p class="card__paragraph">
+                  The lightweight frame that has earned its
+                  street tread as a sleek, agile ride.
+                </p>
+                <span class="card__price">$ 2 090</span>
+              </article>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section
+        class="page__section details"
+        id="details"
+      >
+        <div class="container">
+          <h2 class="page__section-title details__title">
+            The details
+          </h2>
+
+          <article class="details__parts parts" >
+            <div class="parts__images">
+              <div class="grid">
+                <div
+                  class="grid__item grid__item--desktop-1-8 grid__item--tablet-1-4"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details1.jpg"
+                      alt="part1"
```

Comment:
The alt text for the image should be more descriptive, like 'Auto Unlock feature'